SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -1,50 +1,66 @@
-# Get started: 2 - Get and build the GDK’s Unreal Engine Fork
+# Get started: 2 - Get and build the SpatialOS Unreal Engine fork
 
-As the GDK extends Unreal Engine's networking capabilities at its core, to use it you first need to build our SpatialOS fork of Unreal Engine 4 from source.
+The GDK for Unreal extends Unreal Engine's networking capabilities at its core. To use it you first need to build our SpatialOS fork of Unreal Engine from source.
 
-### Unreal Engine EULA
+### Step 1: Unreal Engine EULA
 
-In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
```

Comment:
So, we originally didn't have "accept the invite to join" but someone got trolled by not realising they would get an email invite to join, so we added this in. I can't remember who it was.

Reply:
Good shout. I thought the Unreal documentation mentioned the org invite, but it doesn't seem to.